jaguar-gpu/jaguar-gpu.c

```diff
@@ -1,210 +1,282 @@
-/*
- * Disassembler for the Jaguar GPU/DSP
- * Copyright (C) 2018 - Sebastien Alaiwan
- */
+// SPDX-FileCopyrightText: 2018 Sebastien Alaiwan
+// SPDX-License-Identifier: LGPL-3.0-only
 
-#include <r_asm.h>
-#include <r_lib.h>
+//  Disassembler for the Jaguar GPU/DSP
 
-static int read16_BE (const ut8** b)
-{
+#include <rz_asm.h>
+#include <rz_lib.h>
+
+static int read16_BE(const ut8 **b) {
```

Comment:
Please use endian functions from RzUtil.

Reply:
Could you be a bit more specific on where to look for them?

Reply:
`rz_endian.h`

Reply:
Regarding that, I tried using `rz_read_be16()` but doesn't seem to work.
I think, it's maybe because there's a `(*b) += 2;` over here on this one.

Do you have a pointer to get around this problem?

Reply:
https://www.geeksforgeeks.org/double-pointer-pointer-pointer-c/

Reply:
Thank you! 
Please take a look now. Worked fine with my[ test file](https://www.romnation.net/srv/roms/11071/atarijaguar/Arkanna-Demo-PD.html) locally.